sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/ScriptTransformation.scala

```diff
@@ -58,6 +59,7 @@ case class ScriptTransformation(
     child.execute().mapPartitions { iter =>
       val cmd = List("/bin/bash", "-c", script)
       val builder = new ProcessBuilder(cmd)
+      builder.redirectError(Redirect.INHERIT)
```

Comment:
`builder.redirectError(Redirect.INHERIT)` would consuming the error output from buffer and then print it to stderr (inherit the target from the current Scala process). 
If without this there would be 2 issues:
1) The error msg generated by the script process would be hidden.
2) If the error msg is too big to be hold by the buffer, the input logic would be hang (There's are simple steps on jira to reproduce this behavior).